python/lsst/afw/image/utils.py

```diff
@@ -50,12 +52,16 @@ def clipImage(im, minClip, maxClip):
             mi.getImage(), ds.getFootprints(), maxClip)
 
 
+@deprecated(reason=("Removed with no replacement (FilterLabels do not need to be reset)."
+                    " Will be removed after v23."), category=FutureWarning)
 def resetFilters():
     """Reset registry of filters and filter properties"""
     Filter.reset()
     FilterProperty.reset()
 
 
+@deprecated(reason=("Removed with no replacement (but see lsst::afw::image::TransmissionCurve)."
+                    " Will be removed after v23."), category=FutureWarning)
 def defineFilter(name, lambdaEff, lambdaMin=np.nan, lambdaMax=np.nan, alias=[], force=False):
     """Define a filter and its properties in the filter registry"""
```

Comment:
I'm not sure `TransmissionCurve` is an even approximate analogue for this functionality. This function is fundamentally about editing a central registry so that you can get the info from `Filter` later (AFAIK it's only being called by the obs packages and by unit tests). `TransmissionCurve` is only useful when you're trying to _retrieve_ transmission info, which is not something you do during obs package setup.

Reply:
I guess I was thinking of the wavelengths that get passed in here. I've tweaked it.